kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/internal/linux/executor/ExecutorUtil.java

```diff
@@ -278,15 +295,19 @@ private static DefaultExecutor configureExecutor(Command command) {
         return executor;
     }
 
-    private static void executeAsync(Command command, CommandLine commandLine, Consumer<CommandStatus> callback) {
+    protected Executor getExecutor() {
```

Comment:
Does this need to be protected or maybe better private?

Reply:
I used it only for injecting a mock for testing. I wasn't able to make it work when configured as private.